aiogram_dialog/widgets/kbd/datecalendar.py

```diff
@@ -0,0 +1,163 @@
+from datetime import date, datetime
+from time import mktime
+from calendar import monthcalendar
+from typing import List, Callable, Union, Awaitable
+
+from aiogram.types import InlineKeyboardButton, CallbackQuery
+
+from aiogram_dialog.dialog import Dialog
+from aiogram_dialog.manager.manager import DialogManager
+from aiogram_dialog.manager.intent import ChatEvent
+from aiogram_dialog.widgets.widget_event import WidgetEventProcessor, ensure_event_processor
+from .base import Keyboard
+
+OnDateSelected = Callable[[ChatEvent, "MonthCalendar", DialogManager, date], Awaitable]
+
+LEVEL = "LEVEL"
+YEARS_LEVEL = "Y"
+MONTHS_LEVEL = "M"
+YEAR_PREFIX = "YEAR:"
+MONTH_PREFIX = "MONTH:"
+NEXT_MONTH = "+"
+PREV_MONTH = "-"
+OFFSET = "OFFSET"
+MONTHS_LIST = [range(i, i + 3) for i in range(1, 13, 3)]
```

Comment:
Help me decide, what is cleaner
`[range(i, i + 3) for i in range(1, 13, 3)]`
`[(1, 2, 3), (4, 5, 6), (7, 8, 9), (10, 11, 12)]`

Also, variable is called `list of month`, but it is `list of month ranges`. I cannot understand why it should be list, but not tuple or other collection. Is it so important to be a part of name?